zipkin-ui/js/component_data/default.js

```diff
@@ -6,8 +6,31 @@ import {traceSummary, traceSummariesToMustache} from '../component_ui/traceSumma
 import {mergeV2ById} from '../spanCleaner';
 import {SPAN_V1} from '../spanConverter';
 
+export function convertDurationToMicrosecond(duration) {
+  const match = duration.match(/^(\d+)(μs|ms|s)$/i);
+  if (match) {
+    const unit = match[2];
+    switch (unit) {
+      case 'μs':
```

Comment:
Not that I imagine I'll be using microsecond units, as not having to use that was the point of the change, but I think many people don't know how to easily type `μ` on their keyboard. What if we also accept `us` for microseconds?

Reply:
good point.. I think lack of suffix should be μs. Is it not already the case? good suggestion on us as well.

Reply:
yes, lack of suffix will be μs now.
but I think `us` is also good.
I think we should support both of them.
I'll make another PR.